doc/cephfs/fs-volumes.rst

```diff
@@ -790,32 +790,38 @@ pinned to one of the available ranks on the file system.
-The task arises often in an orchestrated environment such as Kubernetes, where a single deployed application
-can work with many mounted subvolumes across several hosts. When a snapshot of such a system is needed,
-the application may not find the result consistent unless the snapshots were taken
-during an active write pause.
+Distributed applications that run against different mount points of the same Ceph file system
+are facing a challenge of consistent snapshots for backups and disaster recovery. The challenge is due to
+the fact that CephFS snapshots dont provide strong consistency guarantees when there are multiple
+clients performing writes. Even when the application is using file system flushes to synchronize
+checkpoints across its distributed components, there aren't guarantees that all acked writes will
+be part of a given snapshot.
 
-The `volumes` plugin provides a tool to initiate and await such a pause across a set of subvolumes:
+The subvolume quiesce feature has been developed to provide enterprise-level consistency guarantees
+for multi-client applications working with one or more subvolumes. The feature allows to pause IO
+to a set of subvolumes of a given volume (file system). By reaching such a pause across all clients
+we can guarantee that any persistent checkpoints reached by the application will be recoverable
+from the snapshot group made during the pause. We refer to the pause as `quiesce`
+
+The `volumes` plugin now provides a tool to initiate and await the pause for a set of subvolumes:
 
 .. prompt:: bash $ auto
 
   $ ceph fs quiesce <vol_name> --set-id myset1 <[group_name/]sub_name...> --await
   # perform actions while the IO pause is active, like taking snapshots
   $ ceph fs quiesce <vol_name> --set-id myset1 --release --await
-  # if successful, all members of the set were confirmed as still in pause and released from such
+  # if successful, all members of the set were confirmed as still paused and released
 
-The ``quiesce`` functionality is itself based on a lower level QuiesceDb service provided by the MDS
+The ``quiesce`` functionality is based on a lower level QuiesceDb service provided by the MDS
 daemons, which operates at a file system path granularity. 
 The `volumes` plugin merely maps the subvolume names to their corresponding paths on the given file system
 and then issues the appropriate quiesce command to the MDS. You can learn more about the feature in the developer guides.
 
 Operations
 ~~~~~~~~~~
 
-The IO pause (referred to as `quiesce`) is requested for a group of one or more subvolumes (i.e. paths in a filesystem).
-The group is referred to as "quiesce set", and every quiesce set must have a unique string id to interact with.
+The quiesce can be requested for a set of one or more subvolumes (i.e. paths in a filesystem).
+The group is referred to as `quiesce set`, and every quiesce set is identified by a unique `set id`.
```

Comment:
s/The group/This set/

In cases like this (where I am ignorant of the brand-new technology), I think it's good to stick with the noun that was originally introduced. I always wonder in contexts like this if "group" means the same thing as "set". I think that changing "The group" to "This set" (that is, changing the definite article "The" to the demonstrative pronoun "This" makes it crystal clear that we're using this new sentence to talk about the set of subvolumes that we mentioned in the previous sentence.

Reply:
done